src/demo-app/system-config.ts

```diff
@@ -17,36 +17,69 @@ System.config({
     '@angular/compiler': 'node:@angular/compiler/bundles/compiler.umd.js',
     '@angular/http': 'node:@angular/http/bundles/http.umd.js',
     '@angular/forms': 'node:@angular/forms/bundles/forms.umd.js',
-    '@angular/router': 'node:@angular/router/bundles/router.umd.js',
     '@angular/animations': 'node:@angular/animations/bundles/animations.umd.js',
+    '@angular/router': 'node:@angular/router/bundles/router.umd.js',
     '@angular/animations/browser': 'node:@angular/animations/bundles/animations-browser.umd.js',
-    '@angular/platform-browser': 'node:@angular/platform-browser/bundles/platform-browser.umd.js',
     '@angular/platform-browser/animations':
-      'node:@angular/platform-browser/bundles/platform-browser-animations.umd.js',
+      'node:@angular/platform-browser/bundles/platform-browser-animations.umd',
+    '@angular/platform-browser':
+      'node:@angular/platform-browser/bundles/platform-browser.umd.js',
     '@angular/platform-browser-dynamic':
       'node:@angular/platform-browser-dynamic/bundles/platform-browser-dynamic.umd.js',
 
-    '@angular/material': 'dist/bundles/material.umd.js',
-    '@angular/material-moment-adapter': 'dist/bundles/material-moment-adapter.umd.js',
-    '@angular/cdk': 'dist/bundles/cdk.umd.js',
-    '@angular/cdk/a11y': 'dist/bundles/cdk-a11y.umd.js',
-    '@angular/cdk/bidi': 'dist/bundles/cdk-bidi.umd.js',
-    '@angular/cdk/coercion': 'dist/bundles/cdk-coercion.umd.js',
-    '@angular/cdk/collections': 'dist/bundles/cdk-collections.umd.js',
-    '@angular/cdk/keycodes': 'dist/bundles/cdk-keycodes.umd.js',
-    '@angular/cdk/observers': 'dist/bundles/cdk-observers.umd.js',
-    '@angular/cdk/overlay': 'dist/bundles/cdk-overlay.umd.js',
-    '@angular/cdk/platform': 'dist/bundles/cdk-platform.umd.js',
-    '@angular/cdk/portal': 'dist/bundles/cdk-portal.umd.js',
-    '@angular/cdk/rxjs': 'dist/bundles/cdk-rxjs.umd.js',
-    '@angular/cdk/scrolling': 'dist/bundles/cdk-scrolling.umd.js',
-    '@angular/cdk/stepper': 'dist/bundles/cdk-stepper.umd.js',
-    '@angular/cdk/table': 'dist/bundles/cdk-table.umd.js',
-    '@angular/cdk/testing': 'dist/bundles/cdk-testing.umd.js',
+    // TODO(devversion): replace once the index.ts file for the Material package has been added.
+    '@angular/material': 'dist/packages/material/public_api.js',
+    '@angular/cdk': 'dist/packages/cdk/index.js',
```

Comment:
I haven't tried it out, but you should be able to replace these mappings with something like this in the `paths` config:

```
'@angular/cdk/*': 'dist/packages/cdk/*/index.js'
```

Reply:
Unfortunately wildcards are not supported for the `map` option in SystemJS. In the future I hope we can use something like [that](https://github.com/systemjs/systemjs/blob/master/docs/config-api.md#packageconfigpaths)